scrape.sh

```diff
@@ -0,0 +1,8 @@
+if [ -f .env ]; then
```

Comment:
Please add comments to this file.  Specifically, explain what each part is doing, and include links to upstream docs that help give more context.

Reply:
gottcha, basically if there is no .env file it will pull it from the secrets set on github
But if you want to actually build the index you create your own if the site is not deployed yet.

Reply:
![image](https://user-images.githubusercontent.com/55400510/141332146-67ab968a-23e3-4bc1-8039-e996fcdedc9c.png)